src/lib.rs

```diff
@@ -232,6 +232,8 @@ macro_rules! ne_vec {
 
 #[cfg(test)]
 mod tests {
+    use super::*;
```

Comment:
I'm not sure why, but with this line, there's `unused_imports` warning, and the test runs ok without it.

Reply:
When you run the tests without serde, it's unused. I generally use cargo-test-all-features in my projects to make sure everything is covered.